packages/aws-lambda/layer/bin/publish-layer.sh

```diff
@@ -27,24 +27,26 @@ EOF
 }
 
 PACKAGE_NAMES="@instana/aws-lambda instana-aws-lambda-auto-wrap"
-LAYER_NAME=instana-nodejs
+LAYER_NAME=hmadison-instana-nodejs
 LICENSE=MIT
-ZIP_PREFIX=instana-nodejs-layer
+ZIP_PREFIX=hmadison-instana-nodejs-layer
```

Comment:
Just as a reminder for ourselves, this needs to be reverted at some point, shortly before merging this.

Reply:
I updated the script to change the name/region list based off of the `BUILD_NUMBER` environment variable Jenkins sets.